grpc/src/compiler/go_generator.cc

```diff
@@ -187,30 +186,29 @@ void GenerateServerMethod(const grpc_generator::Method *method, grpc_generator::
 		printer->Outdent();
 		printer->Print("}\n\n");
 	}
-
 }
 
 // Generates Client method signature source
 void GenerateClientMethodSignature(const grpc_generator::Method *method, grpc_generator::Printer *printer,
-                                   std::map<grpc::string, grpc::string> vars) {
+                                   std::map<grpc::string, grpc::string> vars, const bool is_func) {
```

Comment:
Is it really needed to add is_func, as it limits the use of signature in future updates.